docs/conceptual/Bar-Memory.rst

```diff
@@ -0,0 +1,98 @@
+.. meta::
+   :description: Learn about BAR memory and ways to handle physical addressing limit in ROCm
+   :keywords: BAR memory, MMIO, GPU memory, Physical Addressing Limit, AMD, ROCm
+
+******************************
+BAR memory overview
+******************************
+From the BIOS of an Intel Xeon E5-based system, you can enable above 4 GB PCIe addressing. You also need to set the
+memory-mapped input/output (MMIO) High Base address ``MMIOH Base`` and range ``MMIO High Size`` in the BIOS.
+
+In the Supermicro system, under the system BIOS, go to ``Advanced > PCIe/PCI/PnP configuration``. You should see the following:
+
+* Above 4G Decoding = Enabled
+* MMIOH Base = 512G
+* MMIO High Size = 256G
+
+.. Note:: 
+
+  When we support the Large Base Address Register (BAR) Capability there is a Large BAR VBIOS that disables the IO BAR.
```

Comment:
The "when we support" feels weird. Either we support it or we don't. Or if we only support it in certain cases, we should say which. And we should also explain what the implications are of disabling the I/O BAR